DashKit/PumpManagerAlert.swift

```diff
@@ -64,8 +64,12 @@ public enum PumpManagerAlert: Hashable {
             return LocalizedString("Auto Off Alert", comment: "Alert content body for autoOff pod alert")
         case .multiCommand:
             return LocalizedString("Multiple Command Alert", comment: "Alert content body for multiCommand pod alert")
-        case .userPodExpiration:
-            return LocalizedString("Pod Expiration Reminder", comment: "Alert content body for userPodExpiration pod alert")
+        case .userPodExpiration(let offset):
+            let formatter = DateComponentsFormatter()
```

Comment:
[super-nit] I know sometimes these `Formatter`s can be expensive to construct, consider keeping around a `lazy static` for reuse?  Not really a huge deal, but thought I'd mention it.

Reply:
It's more the DateFormatter than DateComponentsFormatter, iirc.  I agree it's good to think about that cost; here is a rare usage (once every 3 days). The situation that I normally suggest making formatters less local is in view components that might be rendered often, such as in lists. Static alloc should probably be reserved for very wide scope usage, since once they're allocated, they never go away.